src/main/java/com/example/medicare_call/repository/MemberElderRepository.java

```diff
@@ -15,4 +18,7 @@ public interface MemberElderRepository extends JpaRepository<MemberElder, Long>
     List<MemberElder> findByElder_Id(Integer elderId);
 
     List<MemberElder> findByElder(Elder elder);
+
+    @Query("select me.elder from MemberElder me where me.guardian.id = :guardianId and me.elder.status = :status")
+    List<Elder> findEldersByGuardianIdAndStatus(@Param("guardianId") Integer guardianId, @Param("status") ElderStatus status);
```

Comment:
어플리케이션 계층에 ElderStatus 를 노출하기 보다는 감추는게 좋을 것 같아 보여요
```suggestion
    @Query("select me.elder from MemberElder me where me.guardian.id = :guardianId and me.elder.status = :status")
    List<Elder> findElders(@Param("guardianId") Integer guardianId, @Param("status") ElderStatus status);
    
    default List<Elder> findEldersByGuardianId(Integer guardianId) {
        return findElders(guardianId, ElderStatus.ACTIVE); 
    }
```
는 어떨까요? 의견이 궁금하네요!